### DISCUSSION TITLE
Backtrack e_track=0,tagged_precipitation=0

### DISCUSSION THREAD

**Discussion by YiyingWang-00** [WAM2layers/WAM2layers#202] [Q&A] [answered]
First, I changed the code and the config file according to the comment @rampting in discussion.I can get the right preprocessing data. But, in backtrack, the result e_track=0 and tagged_precipitation=0. This is my config file and running process.

**Comment by YiyingWang-00** [comment]
![微信图片_20230419232013](https://user-images.githubusercontent.com/112865233/233124449-e0aaef77-711e-4bdc-91cc-98cf1e97bda8.png)
![微信图片_20230419232527](https://user-images.githubusercontent.com/112865233/233124459-d137b552-05fd-48bd-a9d1-e5902e4cb261.png)
![微信图片_20230419232532](https://user-images.githubusercontent.com/112865233/233124468-c1faeb2c-b141-4508-8fec-02294343f074.png)

**Reply by ruudvdent** [reply, marked as answer]
It could be that your issue is that the event start date is after the event end date, which should be satisfied (even with backtracking). Yes, backtracking is confusing. A test has been added to the latest release to check these conditions.

**Reply by YiyingWang-00** [reply]
Thank you very much! I have solved the problem by renewing the latest code!

**Comment by YiyingWang-00** [comment]
What is the source region meaning? I create a source_region.nc file by using Matlab, the big region is 60-150E, 10-60N, and the value is 1 in the China region, in other regions the value is 0, is this true?
![微信图片_20230419233211](https://user-images.githubusercontent.com/112865233/233126448-72ae06d8-b8e3-4c29-a375-9ee4e4caebe7.png) @Peter9192 Is this source region caused the wrong in backtrack or other define in my config file? I really need your help, thank you!
@Peter9192

**Reply by ruudvdent** [reply]
Source region is indeed a netcdf containing zeros and ones, so that should not be the issue

**Reply by Peter9192** [reply]
The source region is used to designate the area of interest, i.e. only precipitation over that area is tracked. See https://wam2layers.readthedocs.io/en/latest/userguide.html#tracking-region

**Comment by YiyingWang-00** [comment]
Another question is why the output frequency is 3h, but the output file is only one and one layer,not 8 time layers,like 300 *200 *8?

**Reply by ruudvdent** [reply]
Please use separate questions in separate posts. Could you be a bit more specific what the problem is?
